src/complex/Utilities/Parsing/HDF5/H5Support.hpp

```diff
@@ -79,36 +79,14 @@ namespace Support
  * @param objectName The name of the object to check
  * @return True if the given hdf5 object identifier is a group
  */
-inline bool COMPLEX_EXPORT IsGroup(hid_t nodeId, const std::string& objectName)
-{
-  H5SUPPORT_MUTEX_LOCK()
-
-  bool isGroup = true;
-  herr_t error = -1;
-  H5O_info_t objectInfo{};
-  error = H5Oget_info_by_name(nodeId, objectName.c_str(), &objectInfo, H5P_DEFAULT);
-  if(error < 0)
-  {
-    std::cout << "Error in methd H5Gget_objinfo" << std::endl;
-    return false;
-  }
-  switch(objectInfo.type)
-  {
-  case H5O_TYPE_GROUP:
-    isGroup = true;
-    break;
-  case H5O_TYPE_DATASET:
-    isGroup = false;
-    break;
-  case H5O_TYPE_NAMED_DATATYPE:
-    isGroup = false;
-    break;
-  default:
-    isGroup = false;
-  }
-  return isGroup;
-}
+inline bool COMPLEX_EXPORT IsGroup(hid_t nodeId, const std::string& objectName);
```

Comment:
```suggestion
bool COMPLEX_EXPORT IsGroup(hid_t nodeId, const std::string& objectName);
```
Inline or forward declare functions.